mitodl/ocw-hugo-themes
# move corporate sponsor logos to the theme

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-hugo-themes/issues/106

#### What's this PR do?
With the transition to having `ocw-www`'s content managed by `ocw-studio`, certain static resources that are not editable by a content editor need to be moved over to the theme,  This PR moves the corporate sponsor logos over from `ocw-www` to `ocw-hugo-themes`.  As part of this, the logos for Accenture, Dow and Lockheed were removed per a request from OCW.  The styles were adjusted slightly to accommodate for there only being 3 logos now.

#### How should this be manually tested?
 - Read the readme if you have not run `ocw-www` locally before
 - Clone `ocw-www` and check out the `cg/remove-sponsor-logos` branch, pointing your `ocw-hugo-themes` at it using the `.env` file
 - Spin up the site with `npm start`
 - View the corporate sponsor logo section at the bottom of the page and ensure that it displays correctly with the above mentioned logos removed

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/12089658/121736890-2b140600-cac6-11eb-8a97-d40fe2832422.png)
![image](https://user-images.githubusercontent.com/12089658/121736237-4c282700-cac5-11eb-9db6-4eb647a8beea.png)
